### PR TITLE
Better telemetry for languages

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

**Review by DavidKutu — APPROVED** [review]

File: src/client/datascience/constants.ts [hunk 1]
```diff
@@ -26,6 +26,10 @@ export const KnownKernelLanguageAliases = new Map<string, string>([
     ['c++12', 'c++'],
     ['c++14', 'c++']
 ]);
+/**
+ * This will get updated with the list of VS Code languages.
+ * This way, we can send those via telemetry, instead of having to hardcode the languages.
+ */
```
Comment (IanMatthewHuff) [inline src/client/datascience/constants.ts:32]: The comment here helps, but this seems a bit strange. It's a const array in our datascience/constants file. I don't think that I would expect for anything in this file to be changing dynamically.

Comment (IanMatthewHuff) [reply src/client/datascience/constants.ts:32]: I guess I can't really think of a better location for this off hand. So approving. Just doesn't seem to fit with the rest of this class.

**Review by IanMatthewHuff — APPROVED** [review]

File: src/client/datascience/constants.ts (hunk 1 above)
Comment (DonJayamanne) [reply src/client/datascience/constants.ts:32]: Didn't want to change it a `let xyz`, then it means `xyz` can be updated.

Comment (IanMatthewHuff) [reply src/client/datascience/constants.ts:32]: Yeah, I get that we can add or remove items from it and that it should not be a let. Just more meant that nothing else in this file is getting dynamically updated. If I was using an array from this file I'd be surprised if it was dynamically changed at runtime. I think it works though.

Comment (DonJayamanne) [reply src/client/datascience/constants.ts:32]: Happy to change it to something else, or move it out of the `constants.ts` file if that works.

**Pull request merged**